### ISSUE TITLE
Files containing only interfaces should not be transpiled

### ISSUE THREAD

**Issue by TripleFrequency** [roblox-ts/roblox-ts#88]
After #79 is closed, there will still be another issue to fix:
Files that contain only interfaces are still transpiled.

The only thing these files contain is the TS runtime header.

So,
```TypeScript
export interface A { }
```
Is transpiled as such:
```Lua
-- luacheck: ignore
local TS = require(game:GetService("ReplicatedStorage").RobloxTS.Include.RuntimeLib);
return nil;
```

**Comment by osyrisrblx** [comment]
Why not just use a `.d.ts` in this case? That would make sure you _never_ have non-type code in the file.

**Comment by TripleFrequency** [comment]
Ah, I hadn't thought of that. I agree it is the superior approach, but I think looking into possible fixes for this are worth it, since my example syntax is in no way *invalid*.

I can't think of any *good* approaches to fixing this currently, and really it is no priority, so we can close it if you would like.

**Comment by osyrisrblx** [comment]
> I think looking into possible fixes for this are worth it, since my example syntax is in no way invalid
While the TS code isn't invalid, neither is the Lua code.

..so does it matter?
